magento/magento2
# Fix issue #26276 with clonning quote billing address street

### Description (*)

Fixed cloning billing address to do it recursively including the street array.


### Fixed Issues (if relevant)

1. magento/magento2#26276: Checkout. Quote Address Street cloning issue

### Manual testing scenarios (*)

1. Register an customer account
2. Add an address and mark it **as billing and shipping**
3. Go to the **second** step of checkout
4. Run the following code in the developer console

        # Output and remember what is shipping address city and street
        require('Magento_Checkout/js/model/quote').shippingAddress().city
        require('Magento_Checkout/js/model/quote').shippingAddress().street[0]

        # change the billing address City and Street
        require('Magento_Checkout/js/model/quote').billingAddress().city = "Test"
        require('Magento_Checkout/js/model/quote').billingAddress().street[0] = "123 Main street"

        # Check if shipping address city and street are changed (they should not change after the fix)
        require('Magento_Checkout/js/model/quote').shippingAddress().city
        require('Magento_Checkout/js/model/quote').shippingAddress().street[0]

### Contribution checklist (*)
